test/client/docs_test.rb

```diff
@@ -582,103 +601,113 @@
   it "provides access to multi term vector statistics with .multi_term_vectors" do
     populate!
 
-    response = @docs.multi_term_vectors({ids: [1, 2]}, type: "doc2", fields: "title", term_statistics: true)
+    response = $client.version_support.es_version_7_plus? ? @docs.multi_term_vectors({ids: [1, 2]}, fields: "title", term_statistics: true) : @docs.multi_term_vectors({ids: [1, 2]}, type: "book", fields: "title", term_statistics: true)
     docs = response["docs"]
 
     assert docs
     assert_equal(%w[1 2], docs.map { |h| h["_id"] }.sort)
   end
 
   it "percolates a given document" do
-    populate!
+    if !$client.version_support.es_version_7_plus?
```

Comment:
Same comment here as the other one regarding using `if/skip/end` rather than wrapping so that the test runner sees a skip. Can you update this one as well?

Reply:
Done!